packages/react-components/react-menu/src/contexts/menuTriggerContext.ts

```diff
@@ -1,9 +1,11 @@
 import * as React from 'react';
 
+const newLocal = false;
```

Comment:
Why is this change needed?

Reply:
It's not, definitely a mistake! thx for the input!